### ISSUE TITLE
[Feature]: Clean up signer config

### ISSUE THREAD

**Issue by djordon** [stacks-sbtc/sbtc#429]
# Feature - Clean up signer config

## 1. Description

We should clean up the config to better match the changes we've made in the code base. For example, the block notifier config needs to change, most of those fields are unnecessary. Also, we should choose a different prefix for the environment variables so that there is no clash with the stacks signer. And we need more consistent documentation there as well.

### 2.1 Acceptance Criteria:
   - [ ] Everything is documented.
   - [ ] Nothing there is unnecessary or unused.
   - [ ] It's just better.

**Comment by cylewitruk** [comment]
Recommend `SBTC` instead of `SIGNER`? If multiple services use the same key, I would assume the value should also be the same?

**Comment by djordon** [comment]
I don't think that we are going to change those environment variable names any time soon, although we want to.
